docs/content/en/docs/migrate/strategy/_index.md

```diff
@@ -12,5 +12,259 @@ but the format and content is still evolving.
+When migrating to KLT,
+you need to analyze what each service is doing
+and translate that into an appropriate resource.
+The closest analogy is a Kubernetes
+[orkload](https://kubernetes.io/docs/concepts/workloads/)
+but some services may become
+[Keptn tasks](../../implementing/tasks/)
+or other resources.
+
+### stage
+
+A stage is a subsection of a project
+and has no corresponding KLT component.
+However, the logic of the stages can be useful
+when architecting the migration:
+
+* **deployment stage** -- KLT does not itself deploy the software
+    but rather depends on a deployment engine.
+    However, a `deployment` stage may define sequences of tasks
+    that should be translated into
+    [KeptnTaskDefinition](../../yaml-crd-ref/taskdefinition.md)
+    resources that are executed pre- and post-deployment
+* **testing stage** may define sequences of tasks
+    that should be translated into `KeptnTaskDefinition` resources
+    that are executed pre- and post-deployment.
+
+    Note that all `KeptnTask` resources at the same level
+    (either pre-deployment or post-deployment)
+    execute in parallel
+    whereas Keptn v1 sequences and tasks could execute in parallel.
+    If you have actions that need to execute sequentially,
+    create a single `KeptnTaskDefinition` that calls each action in order.
+    If you have tasks that can execute in parallel,
+    migrating to KLT may improve the performance of the deployment.
+
+### sequence
+
+A sequence is an ordered and user-defined sequence of tasks
+that are defined in a
+[shipyard.yaml](https://keptn.sh/docs/1.0.x/reference/files/shipyard/)
+file.
+The shipyard controller micro-service reads the shipyard file
+and (when the time is right),
+emits a `taskName.triggered` cloud event onto the event bus.
+The shipyard controller waits to receive a `taskName.started` event
+and a correspondingly equal number of `taskName.finished` events
+before the shipyard controller reads the next task
+and emits a `taskName.finished` event for that task.
+
+In this way, you can define arbitrary sequences of any tasks
+at any length and also link (or chain) sequences together
+to form (primitive) workflows.
+When migrating, these sequences of tasks can often be translated into
+[KeptnTaskDefinition](../../yaml-crd-ref/taskdefinition.md)
+resources that are defined to run either pre- or post-deployment
+of the pod-scheduling phase.
+
+The `shipyard` file is a general purpose workflow engine
+that is backed by cloud events.
+It is not opinionated to a tool, platform, technology,
+or a particular "slice" of the lifecycle.
+The `TriggeredOn` property allows
+a Keptn v1 sequence to be triggered at any time
+by a user or another system.
+This capability can be used, for example,
+to trigger a data encryption and backup operation,
+or a file movement over a network, or other arbitrary activities
+that may or may not have anything to do with an application's lifecycle.
```

Comment:
should we shrink this down? maybe instead refer to v1 docs? I feel like this amount of text is unproportioned to the info we give for KLT and some v1 user may find it redundant maybe?

Reply:
Alas, the v1 docs are kind of weak in this area and I'm not going to fix them at this point.  I may, however, move much of this text to the "Paradigm changes" section at the beginning of this page.

Reply:
sounds good, yeah I see no point into enriching v1 docs on this